src/webui/webapplication.cpp

```diff
@@ -735,6 +737,28 @@ void WebApplication::action_command_bottomPrio()
     BitTorrent::Session::instance()->bottomTorrentsPriority(hashes);
 }
 
+void WebApplication::action_command_setLocation()
```

Comment:
if this is asynchronous, what happens if someone calls this in succession/parallel

Reply:
move inherently kind of [checks](https://github.com/Flipkart/qBittorrent/blob/shatabdi/setLocation/src/base/bittorrent/torrenthandle.cpp#L1259), if an existing move already in progress.